rasa_addons/core/tracker_stores/botfront.py

```diff
@@ -249,8 +256,23 @@ def retrieve(self, sender_id):
 
         # the tracker exist localy an there is no new infos
         return self._convert_tracker(sender_id, current_tracker)
-
-    def sweep(self):
+    
+    def cleanup_test_trackers(self):
+        try:
+            for sender_id in list(
+                self.test_trackers.keys()
+            ):
+                max_event_time = time.time() - 600
+                tracker = self.test_trackers.get(sender_id)
+                latest_event = tracker.latest_message.timestamp
+                if latest_event < max_event_time:
+                    logger.debug("SWEEPER: Removing botfront test tracker {}".format(sender_id))
+                    del self.test_trackers[sender_id]
+        except Exception as e:
+            print(e)
+            pass
+    
+    def cleanup_regular_trackers(self):
         try:  ## wraped in a try block so if and exception occurs it does not stopp the sweep mechanism
```

Comment:
the try block is not going to do what it says methinks

Reply:
I've moved it inside the loop which should improve it